src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/common/component/sidebar.yaml

```diff
@@ -10,3 +10,8 @@ sylius_twig_hooks:
             menu:
                 template: '@SyliusAdmin/shared/crud/common/sidebar/menu.html.twig'
                 priority: 0
+
+        'sylius_admin.common.component.sidebar.logo':
+            image:
+                template: '@SyliusAdmin/shared/crud/common/sidebar/logo/image.html.twig'
```

Comment:
I would be for moving the logo template to common directory and use the same one here and in the hook below

Reply:
yeah I agree, just there are two different logos.